### PR TITLE
Update 01-email-system.md (Delivery, Retrieval, grammar...)

### PR DESCRIPTION
**Pull request by SergeCroise**
- Delivery Agent, Retrieval Agent, see https://en.wikipedia.org/wiki/Email_agent_(infrastructure)
- grammar: an email send ... the email will be stored ... (singular)

#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [x] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [ ] 1st Pass (Document is good fit for project and author checklist completed)
- [ ] 2nd Pass (Technical Review - check for technical correctness) 
- [ ] 3rd Pass (Detailed Editorial Review and Peer Review)
- [ ] Final approval (Final Review)

### CODE REVIEW

**Comment by SergeCroise** [comment]
Hello @jimcat8 and @wsoyinka ,
please review!

**Review by jimcat8 — CHANGES_REQUESTED** [review]
!!! question

    Why is DNS required to participate in the work of the email system?

**Review by sspencerwire — APPROVED** [review]

File: docs/guides/email/01-email-system.md [hunk 1]
```diff
@@ -43,12 +42,12 @@ As shown in the picture below, this is a simple illustration of the sending and
...
-MRA: In GNU/Linux, the mainstream and common MDA are **dovecot**
+* **MTA**: In GNU/Linux, the mainstream and common MTA are **postfix**,**qmail**, **sendmail**.
+* **MDA**: In GNU/Linux, the mainstream and common MDA are **procmail** and **maildrop**.
+* **MRA**: In GNU/Linux, the mainstream and common MDA are **dovecot**.
 
-!!! question
-    Why is DNS required to participate in the work of the email system?
+!!! question "Why is DNS required to participate in the work of the email system?"
+    
     In daily use, the standard format for an email user is "username@domain name". A domain name cannot represent a specific host, you need to point the domain name to a specific host name, so we need a DNS MX resolution record.
```
Comment (jimcat8) [inline docs/guides/email/01-email-system.md:51]:
```suggestion
!!! question "Question"
    
    "Why is DNS required to participate in the work of the email system?"
    In daily use, the standard format for an email user is "username@domain name". A domain name cannot represent a specific host, you need to point the domain name to a specific host name, so we need a DNS MX resolution record.
```

**Review by jimcat8 — APPROVED** [review]

**Review by wsoyinka — APPROVED** [review]

**Comment by wsoyinka** [comment]
Thank you @SergeCroise

**Pull request merged**